end-to-end-tests/code-build/README.md

```diff
@@ -0,0 +1,33 @@
+# Setup CodeBuild for End to End Test
+
+We use CodeBuild to run quick end-to-end test. Below are steps to set it up:
+
+### Running `deploy` to setup blox application
+To avoid too broad permissions required for CodeBuild project, we setup blox project firstly.
+
+Make sure you have one AWS credential from the AWS account and used in **default** profile:
+
+```
+$ cat end-to-end-tests/code-build/end2end.properties >> ~/.gradle/gradle.properties
```

Comment:
We probably don't want to make this permanent change. Can we use one of the alternatives outlined in [the docs](https://docs.gradle.org/current/userguide/build_environment.html) to change these gradle properties temporarily?

Reply:
Sure.